go.mod

```diff
@@ -64,6 +64,7 @@ require (
 	google.golang.org/genproto/googleapis/api v0.0.0-20231212172506-995d672761c0
 	google.golang.org/grpc v1.60.1
 	google.golang.org/protobuf v1.32.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Why are there 2 imports for `gopkg.in/yaml.v2 v2.4.0`?

Reply:
Good catch. Was probably duplicated while resolving conflicts with `feature/v0.47.x-ics-lsm`.
Removed by https://github.com/cosmos/cosmos-sdk/pull/19614/commits/987a86dae5f363a744059a4de2e6a9c8dc1e08e0